### ISSUE TITLE
Bioschemas for webpages

### ISSUE THREAD

**Issue by JolandaS** [FAIRplus/FAIRPlus_squad2#72]
Bioschemas implementation to get some basic metadata exposed in web pages for EBiSC

**Comment by AlasdairGray** [comment]
@JolandaS please let @petrosss and myself know what we can do to help this happen. Do we have an idea of what kinds of pages (gene, protein, drugs) will be marked up?

**Comment by JolandaS** [comment]
Thanks for the offer @AlasdairGray. We need to get more details on this ourselves from the data owners. As soon as we have those I will add them here.

**Comment by AlasdairGray** [comment]
@petrosss and myself should both be at the virtual face-to-face so we can help with the hacking then

**Comment by Chris-Evelo** [comment]
Could we think of a use case for this? What would look up this Bioschemas annotation? One idea I can think of is that we could do this for all public IMI dataset descriptions? If so should we create a page that harvests that and tabulates the data descriptions and links. A recipe for the addition of Bioschemas would annotation probably be useful outside the project too. But I guess that is already available, although maybe in another format?

**Comment by AlasdairGray** [comment]
Schema markup of dataset descriptions are already indexed by search engines (Microsoft Bing and Google) which then enable better search, and also dedicated search portals like Google's Dataset Search.

The idea behind including more specific markup on the types themselves is that it also becomes available to the search engines, and also to feed into research knowledge graphs.

**Comment by proccaserra** [comment]
@AlasdairGray +1 to that. There is a dedicated section in the Table of Content for the Cookbook so we can create an issue on the FAIRcookbook tracker an assign you to make a generic case.

**Comment by JolandaS** [comment]
Ask EBiSC on their progress of merging their catalogs.
Bioschemas recipe, discuss during squad lead call.

**Comment by Chris-Evelo** [comment]
Of course, it is wonderful if search engines use it. But I still think it would add an additional benefit if we used the Bioschemas annotation also for cataloging ourselves. Apart from another carrot to actually add the information it will also allow us to evaluate Bioschemas annotations ourselves quicker.

**Comment by JolandaS** [comment]
@Chris-Evelo Could you set up a short recipe in which the outline is given of how you could use Bioschemas for cataloging? And include links to the appropriate Bioschema pages? Just so we have start of a recipe on this subject.

**Comment by JolandaS** [comment]
The webpages meant by EBiSC were indeed the EBiSC catalogue and the hPSCreg webpage. They are in the process of merging these together. I will ask on the current status of this.

**Comment by nsjuty** [comment]
The recipe is written and in the cookbook, but it needs 'framing' in the recipe directly. For instance what 'capability' is needed to do this, such as schema.org and JSON-LD, as well as context. This can be captured in a 'higher level' recipe(s) which will be worked on at some point, and will direct to different 'types', for which this will be one instance.
